client/js/verification-portal/pregnancy-resource-center/form.js

```diff
@@ -260,37 +261,37 @@ class VerificationPortalForm extends Component {
 					<Heading text="HOURS" />
 					<Spacer height="50px" />
 					<label>Sunday Hours</label>
-					<Field name="hours[0].open" component={Input} type="time" />
-					<Field name="hours[0].close" component={Input} type="time" />
+					<Field name="hours[0].open" component={Time} placeholder={'Open'} />
```

Comment:
@bhavyaab We should be able to capture the placeholder in the `Time` component based on the field name so that we don't have to repeat it over and over in this file.